### PR TITLE
Fix saving files

### PR DESCRIPTION
**Pull request by Vlad-Dembrovskyi**
It was found that stringti_merged results folder as well as some other folders are missing all or some files. Most probably that started happening after output pattern update needed to save .command.* logs in #251 .

This PR fixes the issues.

To test:
```
git clone https://github.com/TheJacksonLaboratory/splicing-pipelines-nf
cd splicing-pipelines-nf
git fix-saving-files
NXF_VER=20.01.0 nextflow run . -profile ultra_quick_test,docker --test false
```
The job will fail on the last process, but that's expected.

Then check folders:
- `results/star_mapped/count_matrix` for `sample_lst.txt` file;
- `results/star_mapped/stringtie_merge` for all files (all were missing before);
- `results/MultiQC` for `multiqc_data` folder.

### CODE REVIEW

**Review by cgpu — APPROVED** [review]
Looks good to me implementation wise, approving tentatively but please wait until @angarb reviews and approves as well.

**Comment by Vlad-Dembrovskyi** [comment]
Approved by @angarb https://lifebit-biotech.slack.com/archives/G01017Z5P2P/p1631542779001000?thread_ts=1631024711.007300&cid=G01017Z5P2P
merging

**Pull request merged**
